### PR TITLE
Add rounded corners to left sidebar

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Left sidebar now has rounded corners on the right. Also removed some unneeded css classes and styles.

## Related Issue

- Works towards https://github.com/owncloud/web/issues/6280

## Motivation and Context
Redesign

## Screenshots (if appropriate):
Screenshot 2022-02-03 at 17 09 09
Screenshot 2022-02-03 at 17 09 13

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong> https://drone.owncloud.com/owncloud/web/22427/42/1
The following scenarios passed on retry:
- webUISharingExternalToRoot/federationSharing.feature:378

File: packages/web-app-files/src/views/LocationPicker.vue [hunk 1]
```diff
@@ -486,7 +486,6 @@ export default {
   height: auto;
   z-index: 1;
   grid-area: header;
-  background-color: var(--oc-color-background-default);
```
Comment (pascalwengerter) [inline packages/web-app-files/src/views/LocationPicker.vue:489]: Very weird behaviour when moving a file and scrolling (given enough files in the target folder). Row background colors are different for files&folders, also glitchy in the AppBar (which now is transparent)

File: packages/web-app-files/src/components/SideBar/SideBar.vue [hunk 2]
```diff
@@ -330,7 +330,6 @@ export default {
   max-height: 100%;
   display: grid;
   grid-template-rows: 50px 70px 1fr;
-  background-color: var(--oc-color-background-default);
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/SideBar/SideBar.vue:333]: Now the expanding sub-panels create a weird overlay situation during the animation...

**Review by pascalwengerter — COMMENTED** [review]
Not sure if we're there yet, rounded corners look good but the removed background colors create some (new) glitches from what I witness

File: packages/web-app-files/src/views/LocationPicker.vue (hunk 1 above)
Comment (kulmann) [reply packages/web-app-files/src/views/LocationPicker.vue:489]: Files are disabled as a target in copy/move scenarios, that's why they have different colors in the LocationPicker view. Fixed the transparent background of the AppBar.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingAndUpload</strong> https://drone.owncloud.com/owncloud/web/22441/66/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:94

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal2</strong> https://drone.owncloud.com/owncloud/web/22441/60/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:342

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPublic1</strong> https://drone.owncloud.com/owncloud/web/22441/67/1
The following scenarios passed on retry:
- webUISharingPublicBasic/publicLinkCreate.feature:190

**Review by pascalwengerter — APPROVED** [review]
LGTM ☑️

**Pull request merged**